### ISSUE TITLE
pip.conf required with Dockerfile and does NOT exist in repo

### ISSUE THREAD

**Issue by journeyman-msft** [microsoft/cartpole-py#5]
We should probably remove the dependency on pip.conf

**Comment by ForrestTrepte** [comment]
It looks like pip.conf was removed from Dockerfile in commit 6fd4af3177b226f8fa91940caa8289eee051cca3. @journeyman-msft, please close if that fixes this issue.

**Comment by journeyman-msft** [comment]
thank goodness, yes. Make sure the Dockerfile has pip.conf removed and the requirements.txt has the necessary dependencies

**Comment by ForrestTrepte** [comment]
I can confirm that pip.conf was removed from Dockerfile.

However, there is no currently no requirements.txt. I don't understand the nuances involved, but I see that `pip3 install -r requirements.txt` was replaced with an explicit pip install of bonsai-common via the following:
```
RUN pip3 install -U setuptools \
    && pip3 install git+https://github.com/microsoft/bonsai-common \
    && pip3 uninstall -y setuptools
```

I was able to successfully build the container and locally run a simulator from it on my PC.

@journeyman-msft, does that sound good to you?
